tests/tests_enabled.yml

```diff
@@ -20,3 +20,16 @@
     - name: Check file counts in rsyslog.d
       assert:
         that: rsyslog_d_file_count.matched < 3
+
+    - name: Get rsyslog_version_str
+      command: /bin/grep "^# rsyslog version is " /etc/rsyslog.conf
+      register: rsyslog_version_str
+      changed_when: false
+
+    - name: Retrieve rsyslog_Version
+      set_fact:
+        rsyslog_version: "{{ rsyslog_version_str.stdout | regex_search('[0-9]*\\.[0-9]*\\.[0-9]*') }}"
```

Comment:
Why do you need to set a fact? Can't you just set a variable on the task which needs it, i.e. the task below?

Reply:
Note: it is of course not extremely important to have this done properly in a test playbook, but I would still prefer it to set a good example.

Reply:
> set a variable on the task which needs it, i.e. the task below?

Not sure what you mean

Reply:
```yaml
    - name: Check rsyslog_version
      assert:
        that: rsyslog_version | d() and rsyslog_version != "0.0.0"
      vars:
        rsyslog_version: "{{ rsyslog_version_str.stdout | regex_search('[0-9]*\\.[0-9]*\\.[0-9]*') }}"
```

Reply:
cool - I didn't know about that feature

Reply:
Thank you, @pcahyna and @richm.  I pushed the fix following the suggestion by Pavel.